core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -151,6 +151,7 @@ public String toString() {
           .build();
   public static final PropertyKey NETWORK_HOST_RESOLUTION_TIMEOUT_MS =
       new Builder(Name.NETWORK_HOST_RESOLUTION_TIMEOUT_MS)
+              .setAlias(new String[]{"alluxio.network.host.resolution.timeout.ms"})
```

Comment:
please use the 4-space indent
https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent

in your case here, you can align `.setAlias` with `.setDefaultValue`